templates/_github/workflows/dependabot-auto-merge.raw.yml

```diff
@@ -4,19 +4,14 @@
 name: Auto-Merge Dependabot PRs
 
 on:
-  pull_request:
+  pull_request_target:
 
 jobs:
   auto-merge:
+    if: github.actor == 'dependabot[bot]'
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Check if PR should be auto-merged
-        uses: ahmadnassri/action-dependabot-auto-merge@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
```

Comment:
Nope. Checkout is required because without it the `auto-merge.yml` cannot be found.